student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(name, age, classroom)
+    @classroom = classroom
+    super(name, age)
+  end
```

Comment:
- [ ] Kindly take note that Student inherits from the Person class so the order of the argument in `#super` matters.

- [ ] The `name` parameter should be optional.

- [ ] The `parent_permission` should also be present in the constructor so that when creating the student we can check if they're permitted by parents in the real sense.

  ```
  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @classroom = classroom
  end
  ```